plugins/generator-1.18.2/forge-1.18.2/utils/boundingboxes.java.ftl

```diff
@@ -1,60 +1,50 @@
-<#macro makeBoundingBox positiveBoxes negativeBoxes noOffset facing pitchType="floor">
-    return <#if negativeBoxes?size != 0>Shapes.join(</#if>
+<#macro makeBoundingBox positiveBoxes negativeBoxes facing pitchType="floor">
+    <#if negativeBoxes?size != 0>Shapes.join(</#if>
     <@mergeBoxes positiveBoxes, facing, pitchType/>
     <#if negativeBoxes?size != 0>
     , <@mergeBoxes negativeBoxes, facing, pitchType/>, BooleanOp.ONLY_FIRST)</#if>
-    <#if !noOffset>.move(offset.x, offset.y, offset.z)</#if>;
 </#macro>
 
-<#macro checkPitchSupport positiveBoxes negativeBoxes noOffset facing enablePitch>
+<#macro checkPitchSupport positiveBoxes negativeBoxes facing enablePitch>
     <#if enablePitch>
-        switch ((AttachFace) state.getValue(FACE)) {
-            case FLOOR:
-                <@makeBoundingBox positiveBoxes negativeBoxes noOffset facing "floor"/>
-            case CEILING:
-                <@makeBoundingBox positiveBoxes negativeBoxes noOffset facing "ceiling"/>
-            case WALL:
-            default:
-                <@makeBoundingBox positiveBoxes negativeBoxes noOffset facing "wall"/>
-        }
+        switch (state.getValue(FACE)) {
+            case FLOOR -> <@makeBoundingBox positiveBoxes negativeBoxes facing "floor"/>;
+            case CEILING -> <@makeBoundingBox positiveBoxes negativeBoxes facing "ceiling"/>;
+            case WALL -> <@makeBoundingBox positiveBoxes negativeBoxes facing "wall"/>;

```

Comment:
As you removed the default branch the wall case corresponded to, this wall case should be between floor and ceiling. Other than that, the PR looks good.